active/0000-service-workers.md

```diff
@@ -0,0 +1,49 @@
+- Start Date: 2018-03-13
+- RFC PR: (leave this empty)
+
+# Add a Service Worker with minimal asset caching to a default Ember CLI application
+## Summary
+This RFC introduces a Service Worker to the default app blueprint. It will only cache the most essential assets, such as `index.html`, and the vendor and app specific JS and CSS files, it will only respond with those assets if getting the asset over the network fails.
```

Comment:
How would this work for apps that do not use the `index.html` file that was output in the `/dist` folder in production, and instead just point to the assets from a server rendered page? Would the asset cache list be configurable by environment?